src/daemon/netlink.c

```diff
@@ -28,6 +28,17 @@
 
 #define NETLINK_BUFFER 4096
 
+#ifndef DEFAULT_RTNL_BUFSIZE
+#define DEFAULT_RTNL_BUFSIZE	256 * 1024
+#endif
+
+#ifndef RTNL_SND_BUFSIZE
+#define RTNL_SND_BUFSIZE	DEFAULT_RTNL_BUFSIZE
+#endif
+#ifndef RTNL_RCV_BUFSIZE
+#define RTNL_RCV_BUFSIZE	DEFAULT_RTNL_BUFSIZE
+#endif
```

Comment:
actually, i chose this default value arbitrarily ; 
to fit general needs [ of ours ] ; it could be adapted to something else

in the kernel source, the default value of `rmem_default` is
```
#define SKB_TRUESIZE(X) ((X) +                      \
             SKB_DATA_ALIGN(sizeof(struct sk_buff)) +   \
             SKB_DATA_ALIGN(sizeof(struct skb_shared_info)))

#define _SK_MEM_OVERHEAD        SKB_TRUESIZE(256)
#define _SK_MEM_PACKETS         256

#define SK_RMEM_MAX             (_SK_MEM_OVERHEAD * _SK_MEM_PACKETS)

__u32 sysctl_rmem_default __read_mostly = SK_RMEM_MAX;
```

on our system it shows up as ~160k ; 

on my dev machine:
```
net.core.rmem_default = 212992
net.core.wmem_default = 212992
```

i guess it can vary from system to system, depending how the kernel is compiled